pkg/registry/bundle.go

```diff
@@ -22,7 +29,10 @@ func DefaultYAMLDecoder() runtime.Decoder {
 }
 
 func init() {
-	if err := v1beta1.AddToScheme(Scheme); err != nil {
+	if err := apiextensionsv1beta1.AddToScheme(Scheme); err != nil {
+		panic(err)
+	}
+	if err := apiextensionsv1.AddToScheme(Scheme); err != nil {
```

Comment:
`k8s.io/apimachinery/pkg/util/runtime` has a `Must` function that's canonically used for this:

```golang
utilruntime.Must(apiextensionsv1.AddToScheme(Scheme))
```